acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerRunner.java

```diff
@@ -219,4 +205,23 @@ public int metricsPort() {
       return signerConfig.getMetricsPort();
     }
   }
+
+  private Path createTempDirectory(final String prefix) {
+    try {
+      final Path certificateDirectory = Files.createTempDirectory(prefix);
+      FileUtils.forceDeleteOnExit(certificateDirectory.toFile());
+      return certificateDirectory;
+    } catch (IOException e) {
+      throw new RuntimeException("Unable to create temporary directory", e);
+    }
+  }
+
+  public Path createJksCertFile(final TlsCertificateDefinition caTrustStore) {
+    final Path certificateDirectory = createTempDirectory("acceptance-test-jks-cert");
+    return createJksTrustStore(certificateDirectory, caTrustStore);
+  }
+
+  public SignerConfiguration getSignerConfig() {
```

Comment:
does this need to be public, or just protected?

Reply:
this can be protected. done